fe/fe-core/src/main/java/com/starrocks/transaction/TransactionState.java

```diff
@@ -397,6 +396,17 @@ public void setTransactionStatus(TransactionStatus transactionStatus) {
         }
     }
 
+    public void notifyVisible() {
+        // To avoid the method not having to be called repeatedly or in advance, 
+        // the following trigger conditions have been added
+        // 1. the transactionStatus statu must be VISIBLE
+        // 2. this.latch.countDown(); not called before
+        // 3. this.latch can not be null
+        if (transactionStatus == TransactionStatus.VISIBLE && this.latch.getCount() != 0 && this.latch != null) {            
```

Comment:
this.latch != null && this.latch.getCount() != 0

If this.latch is null, a NPE will be raised.

Reply:
Done